### PR TITLE
modified CNAME record information

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #662

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by mypil** [comment]
@pebaum - PR has been copy edited and is ready for final review. This is based on issue #662.

Thank you.

File: Office365-Admin/dns/create-dns-records-at-google-domains.md [hunk 1]
```diff
@@ -139,7 +139,7 @@ When Office 365 finds the correct TXT record, your domain is verified.
     
     ![Click Save](../media/bf496d01-ccbe-4800-95f4-7b2283f2e5f6.png)
   
-## Add the six CNAME records that are required for Office 365
+## Add the five CNAME records that are required for Office 365
 <a name="BKMK_add_CNAME"> </a>
```
Comment (nenonix) [inline Office365-Admin/dns/create-dns-records-at-google-domains.md]:
```suggestion
```

File: Office365-Admin/dns/create-dns-records-at-google-domains.md [hunk 2]
```diff
@@ -172,11 +172,11 @@ When Office 365 finds the correct TXT record, your domain is verified.
...
-5. Add the other five CNAME records.
+5. Add the other four CNAME records.
     
     In the **Custom resource records** section, create a record by using the values from the next row in the table, and then again choose **Add** to complete that record. 
     
-    Repeat this process until you have created all six CNAME records.
+    Repeat this process until you have created all required CNAME records.
     
 ## Add a TXT record for SPF to help prevent email spam
 <a name="BKMK_add_TXT"> </a>
```
Comment (nenonix) [inline Office365-Admin/dns/create-dns-records-at-google-domains.md]:
```suggestion
```

File: Office365-Admin/dns/create-dns-records-at-google-domains.md [hunk 3]
```diff
@@ -139,7 +139,7 @@ When Office 365 finds the correct TXT record, your domain is verified.
     
     ![Click Save](../media/bf496d01-ccbe-4800-95f4-7b2283f2e5f6.png)
   
-## Add the six CNAME records that are required for Office 365
+## Add the five CNAME records that are required for Office 365
 <a name="BKMK_add_CNAME"> </a>
 
 1. To get started, go to your domains page at Google Domains by using [this link](https://domains.google.com/registrar). You'll be prompted to sign in. To do so:
```
Comment (nenonix) [inline Office365-Admin/dns/create-dns-records-at-google-domains.md]:
```suggestion
1. To get started, go to your [Google Domains page] (https://domains.google.com/registrar) and sign in.
```

File: Office365-Admin/dns/create-dns-records-at-google-domains.md [hunk 4]
```diff
@@ -172,11 +172,11 @@ When Office 365 finds the correct TXT record, your domain is verified.
     
     ![Click Add](../media/4a78080a-e0b2-4582-9696-3fe4fea41e91.png)
   
-5. Add the other five CNAME records.
+5. Add the other four CNAME records.
     
     In the **Custom resource records** section, create a record by using the values from the next row in the table, and then again choose **Add** to complete that record. 
     
-    Repeat this process until you have created all six CNAME records.
+    Repeat this process until you have created all required CNAME records.
```
Comment (nenonix) [inline Office365-Admin/dns/create-dns-records-at-google-domains.md]:
```suggestion
    Repeat this process until you have created all of the five required CNAME records.
```

File: Office365-Admin/dns/create-dns-records-at-google-domains.md (hunk 2 above)
Comment (nenonix) [inline Office365-Admin/dns/create-dns-records-at-google-domains.md]:
```suggestion
```

File: Office365-Admin/dns/create-dns-records-at-google-domains.md (hunk 3 above)
Comment (nenonix) [reply Office365-Admin/dns/create-dns-records-at-google-domains.md]: I think the substeps steps 1and 2 under step one are unnecessary, Suggest step one is simply: "To get started, go to your Google Domains page and sign in."

**Comment by MaratMussabekov** [comment]
Dear @nenonix , your recommendations were successfully applied, thank you!

**Pull request merged**